docs/resource-reference.md

```diff
@@ -10,13 +10,219 @@ Packages:
 
 Resource Types:
 
+- [K0sControllerConfig](#k0scontrollerconfig)
+
 - [K0sWorkerConfig](#k0sworkerconfig)
 
 - [K0sWorkerConfigTemplate](#k0sworkerconfigtemplate)
 
 
 
 
+## K0sControllerConfig
+<sup><sup>[↩ Parent](#bootstrapclusterx-k8siov1beta1 )</sup></sup>
+
+
+
+
+
+
+
+
+<table>
+    <thead>
+        <tr>
+            <th>Name</th>
+            <th>Type</th>
+            <th>Description</th>
+            <th>Required</th>
+        </tr>
+    </thead>
+    <tbody><tr>
+      <td><b>apiVersion</b></td>
+      <td>string</td>
+      <td>bootstrap.cluster.x-k8s.io/v1beta1</td>
+      <td>true</td>
+      </tr>
+      <tr>
+      <td><b>kind</b></td>
+      <td>string</td>
+      <td>K0sControllerConfig</td>
+      <td>true</td>
+      </tr>
+      <tr>
+      <td><b><a href="https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.20/#objectmeta-v1-meta">metadata</a></b></td>
+      <td>object</td>
+      <td>Refer to the Kubernetes API documentation for the fields of the `metadata` field.</td>
+      <td>true</td>
+      </tr><tr>
+        <td><b><a href="#k0scontrollerconfigspec">spec</a></b></td>
+        <td>object</td>
+        <td>
+          <br/>
+        </td>
+        <td>false</td>
+      </tr><tr>
+        <td><b><a href="#k0scontrollerconfigstatus">status</a></b></td>
+        <td>object</td>
+        <td>
+          <br/>
+        </td>
+        <td>false</td>
+      </tr></tbody>
+</table>
+
+
+### K0sControllerConfig.spec
+<sup><sup>[↩ Parent](#k0scontrollerconfig)</sup></sup>
+
+
+
+
+
+<table>
+    <thead>
+        <tr>
+            <th>Name</th>
+            <th>Type</th>
+            <th>Description</th>
+            <th>Required</th>
+        </tr>
+    </thead>
+    <tbody><tr>
+        <td><b>args</b></td>
+        <td>[]string</td>
+        <td>
+          Args specifies extra arguments to be passed to k0s worker. See: https://docs.k0sproject.io/stable/advanced/worker-configuration/<br/>
+        </td>
+        <td>false</td>
+      </tr><tr>
+        <td><b>downloadURL</b></td>
+        <td>string</td>
+        <td>
+          DownloadURL specifies the URL to download k0s binary from. If specified the version field is ignored and what ever version is downloaded from the URL is used.<br/>
```

Comment:
Adjustment: 

DownloadURL specifies the URL from which to download the k0s binary. If the version field is specified, it is ignored, and whatever version is downloaded from the URL is used.

Reply:
Done